custom_components/sat/config_flow.py

```
@@ -84,55 +80,100 @@ async def async_step_dhcp(self, discovery_info: DhcpServiceInfo):
+    async def async_step_mosquitto_opentherm(self, _user_input: dict[str, Any] | None = None):
+        """Setup specific to OpenTherm Gateway."""
+        if _user_input is not None:
+            self.data.update(_user_input)
+
+            return await self.async_step_sensors()
 
+        return self.async_show_form(
+            step_id="mosquitto_opentherm",
+            last_step=False,
+            data_schema=vol.Schema({
+                vol.Required(CONF_NAME, default=DEFAULT_NAME): str,
```

Ruff (0.8.0)

`CONF_NAME` may be undefined, or defined from star imports

(F405)